src/test/java/uk/ac/ebi/subs/POSTingSampleToMultipleSubmissionsTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.ac.ebi.subs;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import uk.ac.ebi.subs.data.structures.WrapperObject;
+import uk.ac.ebi.subs.utils.TestUtils;
+
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class POSTingSampleToMultipleSubmissionsTest {
+
+    static PropertiesManager propertiesManager = PropertiesManager.getInstance();
+
+    static String submitterEmail = propertiesManager.getSubmitterEmail();
+    static String teamName = propertiesManager.getTeamName();
+    static String submissionsApiBaseUrl = propertiesManager.getSubmissionsApiBaseUrl();
+    static String samplesApiBaseUrl = propertiesManager.getSamplesApiBaseUrl();
+    static String samplesInSubmissionByIdUrl = propertiesManager.getSamplesInSubmissionByIdUrl();
+
+    static String[] submissionsUrls;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        TestUtils.createNSubmissions(2, submissionsApiBaseUrl, submitterEmail, teamName);
+        submissionsUrls = TestUtils.getNSubmissionsUrlsForTeam(teamName, 2);
```

Comment:
You might not get back the submissionURLs you created 1 line before, if somebody created some submission meanwhile for the same submitter.

Reply:
There's no work around that, that's why the team name should be used as an unique identifier (as stated in the README file).

Reply:
This issue will be difficult to get around...I was thinking that each test could generate a random team name to make collisions like this unlikely, but when AAP is in place we'll need to make sure that the teamName exists as an AAP domain before creating the submission. Not sure how to get around this.

Reply:
The AAP will solve this problem we'll be able to use a domain/team just for this purpose and as such it will be unique and no one without a valid token will be able to create submissions for that team :)

Reply:
It's possible that two devs could run the tests simultaneously with the same token though. Unlikely, but happened to me before! 👍